doc/overrides.md

```diff
@@ -246,6 +246,10 @@ signatures:
 - `i64 @labs( i64 )`
 - `i64 @llabs( i64 )`
 - `i32 @__cxa_atexit( void (i8*)*, i8*, i8* )`
+- `i32 @open( i8*, i32 )`
+- `i32 @close( i32 )`
+- `ssize_t @read( i32, i8*, size_t )`
+- `ssize_t @write( i32, i8*, size_t )`
```

Comment:
I think these [merit additional discussion](https://github.com/GaloisInc/grease/pull/151/files#diff-7dd685a8abdc7af96594c6f1cc945ea58675e9238a0392d269c3256fa50ad329L354), perhaps just a link [here](https://github.com/GaloisInc/crucible/tree/fb06f32e70de358c4affc5f12e54b0cbb248b5ce/crucible-symio) and/or [here](https://github.com/GaloisInc/crucible/tree/master/crux-llvm#symbolic-io-experimental).